plugins/datahub-integration/src/main/resources/config/simplest-default.toml

```diff
@@ -0,0 +1,14 @@
+[global]
+geonetwork4_api_url = "/geonetwork/srv/api"
+datahub_url = "/datahub"
+proxy_path = ""
```

Comment:
I would include the languages option here

Reply:
also contact email (commented out)

Reply:
and favicon (which might be extensively used for custom sub portals)